source/release-notes/1.5.txt

```diff
@@ -0,0 +1,53 @@
+.. _c2c-release-notes-1.5:
+
+===================================
+Release Notes for ``mongosync`` 1.5
```

Comment:
(just a thought)
I know this goes back to 1.0.0, but I'm just noticing that we consistently use 1.x.x , except in the title of the Release Note page. I might be something to be consistent on, unless I've missed a conversation about why it's important to do it this way. Not urgent and not blocking!

Reply:
Fixed

Reply:
Hey @kennethdyer. This is definitely non blocking. But I don't see the fix. Making the suggestion here and feel to accept or reject, but you'd want to do it for each of the release version titles.